pkg/pxc/queries/queries.go

```diff
@@ -246,18 +248,27 @@ func (p *Database) IsReadonly() (bool, error) {
 }
 
 func (p *Database) StartReplication(replicaPass string, config ReplicationConfig) error {
+	var SSL int = 0
+	var CA string = ""
+	if config.SSL {
+		SSL = 1
+		CA = config.CA
+	}
 	_, err := p.db.Exec(`
 	CHANGE REPLICATION SOURCE TO
-    SOURCE_USER='replication',
-    SOURCE_PASSWORD=?,
-    SOURCE_HOST=?,
-	SOURCE_PORT=?,
-    SOURCE_CONNECTION_AUTO_FAILOVER=1,
-	SOURCE_AUTO_POSITION=1,
-    SOURCE_RETRY_COUNT=?,
-    SOURCE_CONNECT_RETRY=?
-    FOR CHANNEL ?
-`, replicaPass, config.Source.Host, config.Source.Port, config.SourceRetryCount, config.SourceConnectRetry, config.Source.Name)
+		SOURCE_USER='replication',
+		SOURCE_PASSWORD=?,
+		SOURCE_HOST=?,
+		SOURCE_PORT=?,
+		SOURCE_CONNECTION_AUTO_FAILOVER=1,
+		SOURCE_AUTO_POSITION=1,
+		SOURCE_RETRY_COUNT=?,
+		SOURCE_CONNECT_RETRY=?,
+		SOURCE_SSL=?,
+		SOURCE_SSL_VERIFY_SERVER_CERT=?,
```

Comment:
@aaguilartablada sorry for asking this after a long time but I think we need to let users configure SSL verification. Adding a new bool field is enough. If you're not able to do it, I'll make the necessary changes